src/Compilers/CSharp/Portable/Syntax/CSharpSyntaxTree.cs

```diff
@@ -624,7 +624,7 @@ internal PragmaWarningState GetPragmaDirectiveWarningState(string id, int positi
         /// `enable` or `safeonly`, false if `disable`, and null if no preceding directive,
```

Comment:
>safeonly [](start = 25, length = 8)

Could you fix or remove this comment as well? `safeonly` is no longer an option. Thanks